numpy/ma/tests/test_core.py

```diff
@@ -2289,135 +2302,104 @@ def test_datafriendly_mul_arrays(self):
     def test_inplace_addition_scalar_type(self):
         # Test of inplace additions
         for t in self.othertypes:
-            with warnings.catch_warnings(record=True) as w:
-                warnings.filterwarnings("always")
-                (x, y, xm) = (_.astype(t) for _ in self.uint8data)
-                xm[2] = masked
-                x += t(1)
-                assert_equal(x, y + t(1))
-                xm += t(1)
-                assert_equal(xm, y + t(1))
-
-                assert_equal(len(w), 0, "Failed on type=%s." % t)
+            (x, y, xm) = (_.astype(t) for _ in self.uint8data)
+            xm[2] = masked
+            x += t(1)
+            assert_equal(x, y + t(1))
+            xm += t(1)
+            assert_equal(xm, y + t(1))
```

Comment:
Hmm, the behavior changes here. Before the test failed if any warnings were raised, that is no longer true.